### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
## PR checklist

- [x ] This comment contains a description of changes (with reason)
- [x ] `CHANGELOG.md` is updated
- [x ] If you've fixed a bug or added code that should be tested, add tests!
- [x ] Documentation in `docs` is updated
- [x ] If necessary, also make a PR on the [nf-core/diaproteomics branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/diaproteomics)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,7 +61,26 @@ jobs:
...
+
+      - name: Build new docker image
+        if: env.GIT_DIFF
+        run: docker build --no-cache . -t nfcore/diaproteomics:1.0.0
+
+      - name: Pull docker image
+        if: ${{ !env.GIT_DIFF }}
+        run: |
+          docker pull nfcore/diaproteomics:dev
+          docker tag nfcore/diaproteomics:dev nfcore/diaproteomics:1.0.0
```
Comment (ggabernet) [inline .github/workflows/ci.yml:82]: Check line 92: there `sarek` container is pulled instead of `diaproteomics`

**Review by ggabernet — COMMENTED** [review]
Just a small fix and ready to go!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Leon-Bichmann) [reply .github/workflows/ci.yml:82]: ha ok thx! I'll fix it.

Comment (Leon-Bichmann) [reply .github/workflows/ci.yml:82]: done

**Comment by Leon-Bichmann** [comment]
Ok I think everything should be ready now. @ggabernet @timosachsenberg plz approve.

**Review by timosachsenberg — APPROVED** [review]
nice

**Review by ggabernet — APPROVED** [review]
Looks good to me!

Let's not forget to create the Zenodo DOI and other necessary things after the first release that are described here: https://nf-co.re/developers/release_checklist

**Pull request merged**